careerfairsystems/nexpo-app
# Update project to make Expo Go work

here was a solution to Expo Go issues!
The project needed to update to Expo 46 and renew some deprecated dependencies to make it work on the Expo Go application. However there are some more deprecated dependencies which needs to be fixed but that is a future issue as the app is still working.

To test this you need to do the following:
- Update/Download the latest version of the Expo Go application
- Make sure you are on the same Wifi/Network on the phone as you are on your computer
- Scan the QR code when you have done npm install and npm run start

Hopefully this works for you as well. Try also with the Backend deployed on your computer and login.

Fixes the issue #267
